test/Microsoft.AspNet.Razor.Runtime.Precompilation.Test/project.json

```diff
@@ -23,6 +23,9 @@
     },
     "frameworks": {
         "dnx451": {
+            "frameworkAssemblies": {
+                "System.Text.Encoding": "4.0.0.0"
```

Comment:
I am about to submit a commit removing the version numbers from all `frameworkAssemblies` entries in Testing. Hopefully that will allow @pranavkm to clean this up too.